extension.html

```diff
@@ -0,0 +1,119 @@
+      </p>
+    </section>
+
+    <section id='sotd'>
+      <p>
+        This proposal was first introduced in this <a href="https://discourse.wicg.io/t/new-event-for-non-primary-button-click/1527">discourse</a>.
+      </p>
+    </section>
+
+    <section id='intro' class='informative'>
+      <h2>Introduction</h2>
+      <p>
+        This document mainly contains the new APIs that may not make it to Pointer Event Level 2 spec. So we iterate over these new APIs to make them ready for the next version.
+      </p>
+      <p>
+	One of the APIs that are added in this document exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps might want to take advantage of the a precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. So we needed an API to expose the coalesced events along with the event that was dispatched to the javascript. 
+      </p>
+    </section>
+    <section dfn-for="extensions-to-pointerevents-interface">
+      <h2>Extensions to the <code>PointerEvent</code> interface</h2>
+      <p>
+        The following section describes extensions to the existing <a href="https://w3c.github.io/pointerevents/index.html#pointerevent-interface"><code>PointerEvent</code> interface</a>.
+      </p>
+      <div>
+          <pre class="idl">
+partial dictionary PointerEventInit {
+    sequence&lt;PointerEventInit> coalescedEventInits = [];
+};
+
+partial interface PointerEvent {
+    sequence&lt;PointerEvent> getCoalescedEvents();
+};
+          </pre>
+          <dl dfn-for="PointerEvent" link-for="PointerEvent">
+              <dt><dfn>getCoalescedEvents</dfn></dt>
+              <dd>
+                  <p> Returns a sequence of all <code>PointerEvents</code> that were coalesced into the received event. The following attributes of the coalesced events will always have the same value as the received event: <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]). So the first event will have the smallest <code>timeStamp</code>. Also the received event's <code>timeStamp</code> will be greater than or equal to the last event's <code>timeStamp</code> in the sequence. None of the events in the sequence will have coalesced events. When an event has no coalesed event, this function returns an empty sequence.</p>
```

Comment:
@RByers, @dtapuska: With coalescing, the time-stamps of the received events may not reflect the times when the events occurred. Is it okay? Consider a single pointermove that occurred at time T, which the browser held back until the next frame timing at T+t1. Assuming for simplicity that there is no other held-back events, is it really okay (or possible) that the single pointermove event fired at time T+t1 would have a timestamp T?